### DISCUSSION TITLE
GitHub Project board is having 100% CPU and freeze forever

### DISCUSSION THREAD

**Discussion by fzyzcjy** [community/community#48878] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hi, thanks for GitHub and I use it everyday for a long time. However, the github project board (https://github.com/fzyzcjy/a-private-repo/projects/a-project-number) is freezing since roughly yesterday. It eats from 30% to 130% CPU when I look at system monitor forever, and even starts to freeze forever once in a while.

More information (not sure whether related): The project board's `Done` column has ~1000 issues. The board has 6 columns, and except for the Done column, each column only has ~10 issues.

---

EDIT: I am not sure why there are two discussions (I only created once...). The other is at: https://github.com/community/community/discussions/48879

**Comment by queenofcorgis** [comment]
Hey @fzyzcjy, I'll mention your issue to some of our Projects folks, but in the immediate I'm a bit curious what browser you used to post this discussion since you mentioned it posted twice.

**Reply by fzyzcjy** [reply]
Hi thanks for the reply. Chrome 110.0.5481.177 at MacOS intel chip

**Reply by fzyzcjy** [reply]
Btw the bug is still there, making project boards really hard to use since it freezes and freezes :(

**Reply by queenofcorgis** [reply]
Are you using the old Projects board (classic) or the new one? I let the team know and that is our first question.

**Reply by fzyzcjy** [reply]
Old (classical)

**Reply by fzyzcjy** [reply]
Btw I like the old projects' GUI ;)

**Reply by 60355322264** [reply]
> Hey@fzyzcjy, Sorununuzdan bazı Proje arkadaşlarımıza bahsedeceğim, ancak hemen bu tartışmayı iki kez yayınladığınızdan bahsettiğiniz için hangi tarayıcıyı kullandığınızı merak ediyorum.

Devre dışı bırakmak istiyorum

**Reply by 60355322264** [reply]
> Merhaba cevap için teşekkürler. MacOS intel çipinde Chrome 110.0.5481.177

Devre dışı bırakmak istiyorum

**Comment by queenofcorgis** [comment]
Going to separate out the thread here so it is clear to any other users who may be encountering a similar situation. 

1. Since you mention you like the old Projects' GUI... the best route would be to archive the contents of your 'Done' column - you can do this in bulk by clicking the three dots on the top of the column you'd like to archive and there is an option for Archive all cards. 
2. I think you're less likely to go this route, but there is the option to migrate over to the new Projects board and we have some [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/migrating-from-projects-classic) on that here.

**Reply by fzyzcjy** [reply]
> the best route would be to archive the contents of your 'Done' column

Thanks, but I hope it is not archived. It is useful to have a look at the column. I know archiving may be solving the problem (with experiments and guesses), but since *a few days ago* the bug has not exist at all, I guess this is avoidable instead of a deeply rooted bug.

> I think you're less likely to go this route,

Your guess is correct ;)

**Reply by queenofcorgis** [reply]
I'm chatting with some Product managers who support the new Projects and we're curious to know if you'd like to share what you like about the GUI in Classic vs the new one.

**Reply by fzyzcjy** [reply]
Sure! I will make a reply after ~8 hours when having time

**Reply by fzyzcjy** [reply]
I will try the new projects first before answering this - hopefully the new projects board evolves after last time I check it

EDIT: Try *again* (since tried before and did not feel that excellent)

**Reply by queenofcorgis** [reply]
That is awesome to hear you'll be giving it a try - we are excited to hear your feedback!

**Reply by fzyzcjy** [reply]
It is much better than my previous (several months / a year ago, not remember exactly) experience! For example, I still remember, when entering text, if I did not tap enter but accidentally tap other key, it will be throw away completely and I have to type again.

**Comment by maciejgliwinski** [comment]
I am encountering the same issue. Everything was working smoothly and last week it stopped working. I am using Chrome (110.0.5481.178). I Network tab of developer tools I can see that the site is spamming with cards fetch requests. Blocking the domain in developer tools fixes the issue (sort of)

**Reply by fzyzcjy** [reply]
>  the site is spamming with cards fetch requests

I can confirm that I was also seeing this
